From f0d7cf0577a743b10a6550f80ac7ac0308d4d8fe Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 10 Apr 2024 11:33:31 +0200
Subject: [PATCH 0815/1695] dts: stm32: set TAMP as a wake up source on
 stm32mp135f-dk board

Set tampers as a wake up source on this board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 79de963bdedd6cf2c45444b54ed0297b5d6b6183 dts: stm32: set TAMP as a wake up source on stm32mp135f-dk board]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ie7cd42463d5db4161915f5ca452d5135cb9ca330
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/372660
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index 60d3741e8..d8e5f0ddf 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -646,6 +646,8 @@
 	st,backup-zones = <10 5 17>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
+	wakeup-source;
+	wakeup-parent = <&exti>;
 
 	/* Tamper button */
 	tamp-button {
-- 
2.47.2

